e2e/transform/transform-snapshotResolver/babel.config.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = {
+  presets: [
+    '@babel/preset-typescript',
+    [
+      '@babel/preset-env',
+      {
+        targets: {node: 6},
```

Comment:
In e2e where this does not affect shipped code, `'current'` instead of `6` might make more sense to avoid having to update it regularly.
(Given that preset-typescript is the one used by the test for demonstrating that the transform is applied correctly)